tokens/zeplin.json

```diff
@@ -0,0 +1,3444 @@
+		},
+		"typography-mobile-body-small-bold-autoheight-left": {
+			"value": {
+				"lineHeight": 20,
+				"font": {
+					"family": "DBScreenSans",
+					"size": 12,
+					"weight": 700,
+					"stretch": 1
+				}
+			},
+			"woff": { "value": "assets/fonts/dbscreensans-bold.woff" },
+			"woff2": { "value": "assets/fonts/dbscreensans-bold.woff2" }
+		},
+		"typography-mobile-body-small-autowidth-center": {
+			"value": {
+				"lineHeight": 20,
+				"alignment": "center",
+				"font": {
+					"family": "DBScreenSans",
+					"size": 12,
+					"weight": 400,
+					"stretch": 1
+				}
+			},
+			"woff": { "value": "assets/fonts/dbscreensans-regular.woff" },
+			"woff2": { "value": "assets/fonts/dbscreensans-regular.woff2" }
+		},
+		"typography-mobile-body-small-autowidth-left": {
+			"value": {
+				"lineHeight": 20,
+				"font": {
+					"family": "DBScreenSans",
+					"size": 12,
+					"weight": 400,
+					"stretch": 1
+				}
+			},
+			"woff": { "value": "assets/fonts/dbscreensans-regular.woff" },
+			"woff2": { "value": "assets/fonts/dbscreensans-regular.woff2" }
+		},
+		"typography-mobile-body-small-autowidth-right": {
+			"value": {
+				"lineHeight": 20,
+				"alignment": "right",
+				"font": {
+					"family": "DBScreenSans",
+					"size": 12,
+					"weight": 400,
+					"stretch": 1
+				}
+			},
+			"woff": { "value": "assets/fonts/dbscreensans-regular.woff" },
+			"woff2": { "value": "assets/fonts/dbscreensans-regular.woff2" }
+		},
+		"typography-mobile-body-small-bold-autoheight-right": {
+			"value": {
+				"lineHeight": 20,
+				"alignment": "right",
+				"font": {
+					"family": "DBScreenSans",
+					"size": 12,
+					"weight": 700,
+					"stretch": 1
+				}
+			},
+			"woff": { "value": "assets/fonts/dbscreensans-bold.woff" },
+			"woff2": { "value": "assets/fonts/dbscreensans-bold.woff2" }
+		},
+		"typography-mobile-body-small-autoheight-center": {
+			"value": {
+				"lineHeight": 20,
+				"alignment": "center",
+				"font": {
+					"family": "DBScreenSans",
+					"size": 12,
+					"weight": 400,
+					"stretch": 1
+				}
+			},
+			"woff": { "value": "assets/fonts/dbscreensans-regular.woff" },
+			"woff2": { "value": "assets/fonts/dbscreensans-regular.woff2" }
+		},
+		"typography-mobile-body-small-autoheight-left": {
+			"value": {
+				"lineHeight": 20,
+				"font": {
+					"family": "DBScreenSans",
+					"size": 12,
+					"weight": 400,
+					"stretch": 1
+				}
+			},
+			"woff": { "value": "assets/fonts/dbscreensans-regular.woff" },
+			"woff2": { "value": "assets/fonts/dbscreensans-regular.woff2" }
+		},
+		"typography-mobile-body-small-autoheight-right": {
+			"value": {
+				"lineHeight": 20,
+				"alignment": "right",
+				"font": {
+					"family": "DBScreenSans",
+					"size": 12,
+					"weight": 400,
+					"stretch": 1
+				}
+			},
+			"woff": { "value": "assets/fonts/dbscreensans-regular.woff" },
+			"woff2": { "value": "assets/fonts/dbscreensans-regular.woff2" }
+		}
+	}
+}
```

Comment:
I think that this even already shows the downside of this procedure: We wouldn't need to include all of those tokens, as only a handful is really necessary.

Reply:
https://github.com/db-ui/base/blob/feat-zeplin-token-generation/scripts/zeplin-styleguide.js#L40

I add a TODO already. You are absolutely right, we need to make it more generic and merge all those. I guess it will only work if we make a commitment with UX on how we format everything. 
As for now the keys are separated by a hyphen so it is pretty easy fo us to merge them.